app/src/main/java/com/riteh/autoshare/adapters/CardListAdapter.kt

```diff
@@ -20,17 +20,26 @@ import kotlinx.coroutines.withContext
 
 class CardListAdapter(private var cards: MutableList<UserCard>): RecyclerView.Adapter<CardListAdapter.ViewHolder>() {
 
+    /**
+     * Involves inflating a layout from XML and returning the holder
+     */
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): CardListAdapter.ViewHolder {
         val view = LayoutInflater.from(parent.context).inflate(R.layout.credit_card_layout, parent, false)
 
         return ViewHolder(view)
     }
 
+    /**
+     * Length list
```

Comment:
Za ubuduće, piše se "Length of list" ili "List length", trebalo bi biti ili "_svojstvo_ of _entitet_" ili  "_entitet svojstvo_"